demos/appyx-navigation/desktop/src/desktopMain/kotlin/com/bumble/appyx/navigation/Main.kt

```diff
@@ -0,0 +1,85 @@
+import androidx.compose.ui.input.key.type
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import com.bumble.appyx.components.backstack.BackStack
+import com.bumble.appyx.components.backstack.BackStackModel
+import com.bumble.appyx.components.backstack.ui.slider.BackStackSlider
+import com.bumble.appyx.navigation.node.container.ContainerNode
+import com.bumble.appyx.navigation.ui.AppyxSampleAppTheme
+import com.bumble.navigation.integrationpoint.DesktopNodeHost
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.mapNotNull
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.launch
+
+sealed class Events {
+    object OnBackPressed : Events()
+}
+
+fun main() = application {
+    val events: Channel<Events> = Channel()
+    val windowState = rememberWindowState(size = DpSize(480.dp, 658.dp))
+    val eventScope = remember { CoroutineScope(SupervisorJob() + Dispatchers.Main) }
+    Window(
+        state = windowState,
+        onCloseRequest = ::exitApplication,
+        onKeyEvent = { onKeyEvent(it, events, eventScope) },
+    ) {
+        AppyxSampleAppTheme {
+            Surface(color = MaterialTheme.colorScheme.background) {
+                Column {
+                    DesktopNodeHost(
+                        windowState = windowState,
+                        onBackPressedEvents = events.receiveAsFlow().mapNotNull {
+                            if (it is Events.OnBackPressed) Unit else null
+                        }
+                    ) { buildContext ->
+                        val backStack: BackStack<ContainerNode.InteractionTarget> = BackStack(
+                            model = BackStackModel(
+                                initialTargets = listOf(ContainerNode.InteractionTarget.Selector),
+                                savedStateMap = buildContext.savedStateMap,
+                            ),
+                            motionController = { BackStackSlider(it) }
+                        )
+                        ContainerNode(
+                            buildContext = buildContext,
+                            backStack = backStack,
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+private fun onKeyEvent(
+    keyEvent: KeyEvent,
+    events: Channel<Events>,
+    coroutineScope: CoroutineScope = CoroutineScope(SupervisorJob() + Dispatchers.Main),
+): Boolean =
+    when {
+        keyEvent.type == KeyEventType.KeyDown && keyEvent.key == Key.Backspace -> {
+            coroutineScope.launch { events.send(Events.OnBackPressed) }
+            true
+        }
```

Comment:
We could consider escape instead of backspace maybe? Should it be configurable by client code?